browser/js/auth/auth.html

```diff
@@ -0,0 +1,25 @@
+<form id="auth-form" name="authForm" ng-submit="authForm.$valid && sendAuth(auth)">
+
+    <div class="modal-header">
+        <h3 class="modal-title"> Login or Signup </h3>
+    </div>
+    <div class="modal-body">
+    <!-- have alert show for empty input as well -->
+        
+        <alert type="danger" ng-show="error">
+            {{ error }}
+        </alert>
+
+        <div class="form-group">
+            <label for="email">Email address</label>
+            <input type="text" ng-model="auth.email" required class="form-control" id="email" placeholder="Enter email">
```

Comment:
right now the form won't check if you enter a valid email. 
If you change 'type' to email you can also check that on the form